docs/ja/user-guide/configuration/index.md

```diff
@@ -15,80 +15,98 @@ toc:
 
 各プロパティ名にマウスカーソルを乗せるとそれらの説明が表示されます。
 
-
 <div class="yaml-docs">
-
+</div>
+<div data-md-type="block_html">
```

Comment:
Probably, this change isn't unnecessary.

Reply:
deleted!